src/connectors/stingray.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+Connector.playerSelector = 'player';
+Connector.artistSelector = 'cover-with-info.current.cover artist-names div.value';
+Connector.trackSelector = 'cover-with-info.current.cover div.title-content > span.title';
+Connector.albumSelector = 'cover-with-info.current.cover div.album';
+Connector.getTrackArt = () => Util.extractUrlFromCssProperty(document.querySelector('cover-with-info.current.cover div.background').style['background-image']);
```

Comment:
Default implementation has already this logic. Using `Connector.trackArtSelector` property is enough here.